doc/blog/Wasm-JS series/Blog - WASM.md

```diff
@@ -0,0 +1,658 @@
+       SetStyle("white-space", "pre-wrap");
+   
+       // Update the display when the element is loaded in the DOM
+       Loaded += (snd, evt) => UpdateDisplay(newLanguage: Language);
+   }
+   ```
+
+5. Compile & run.  It should work like this:
+   ![image-20200415135422628](image-20200415135422628.png)
+
+## 🔬 Going further
+
+This sample is a very simple integration as there is no _callback_ from HTML to managed code and _PrismJS_ is a self-contained framework (it does not download any other javascript dependencies).
+
+Some additional improvements can be done to make the code more production ready:
+
+* **Make the control multi-platform**. A simple way would be to use a WebView on other platforms, giving the exact same text-rendering framework everywhere. The code of this sample won't compile on other targets.
+* **Create script files instead of generating dynamic JavaScript**. That would have the advantage of improving performance and make it easier to debug the code. A few projects are also using TypeScript to generate JavaScript. This approach is done by Uno itself for the `Uno.UI.Wasm` project: https://github.com/unoplatform/uno/tree/master/src/Uno.UI.Wasm.
+* **Support more PrismJS features**. There are many [_plugins_ for PrismJS](https://prismjs.com/#plugins) that can be used. Most of them are very easy to implement.
+
+
+
+# Article 2 - Callback to app from JavaScript
+
+In the previous article, a simple _syntax highlighter_ was used to enhance the display of text in HTML. But it is not enough for most apps: it's often required for JavaScript components to call back into the C# side of application. The easiest way to do that in Uno for WebAssembly is by using [_DOM Events_](https://developer.mozilla.org/docs/Web/Guide/Events/Creating_and_triggering_events). Applications using Uno can [consume DOM Events and CustomEvents](https://platform.uno/docs/articles/wasm-custom-events.html) very easily.
+
+Let's create an application illustrating how to use this feature.
+
+# Sample 2 - Integration of Flatpickr
+
+📝 [Flatpickr](https://flatpickr.js.org/) is a lightweight, self-contained date and time picker. It's an easy way to explore how a JavaScript can call back to the managed application code. In this case, this will be used to report when the picker is opened and a date and time was picked.
+
+## 1. Create the solution in VisualStudio
+
+📝 This part is very short because it is similar to previous article:
+
+1. Create a `Cross-Platform App (Uno Platform)` project and name it `FlatpickrDemo`.
+2. Remove `.Droid`, `.iOS` & `.UWP` projects from solution.
+3. Update to latest _stable_ version of `Uno.*` dependencies.
+
+## 2. Inject Flatpickr from CDN
+
+🎯 This section is using a CDN to get _Flatpickr_ instead of hosting the javascript directly in the application. It is not always the best solution as it creates a dependency on the Internet availability. Any change made server-side could break the application.
```

Comment:
```suggestion
🎯 This section is using a CDN to get _Flatpickr_ instead of hosting the Javascript directly in the application. It is not always the best solution as it creates a dependency on the Internet availability. Any change made server-side could break the application.
```